eng/pipelines/libraries/run-test-job.yml

```diff
@@ -81,6 +85,16 @@ jobs:
                     /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/overrideCoreclrFromLiveDrop.binlog
             displayName: Override $(runtimeFlavorName) from live drop
 
+        - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
```

Comment:
You don't need to duplicate this if condition, you can just use the one above, yml is indentation based.

Reply:
good catch.